### PR TITLE
LPS-173150 Refine the process of importing fragments and page templates

### PR DESCRIPTION
**Pull request by claraizquierdo**
[Link to story or ticket](https://issues.liferay.com/browse/LPS-173150)

### CODE REVIEW

**Comment by claraizquierdo** [comment]
Hi @ealonso , could you help me with the following requirement for both fragments and page templates?

_6. Add alerts (see list of alerts below).
    - Even if there are errors during the import process, we do want to close the modal and show any alert outside of it._

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/view_import.jsp [hunk 1]
```diff
@@ -78,16 +82,17 @@ ImportDisplayContext importDisplayContext = new ImportDisplayContext(request, re
 
 		<liferay-ui:error exception="<%= InvalidFileException.class %>" message="the-selected-file-is-not-a-valid-zip-file" />
 
+		<p><liferay-ui:message key="select-a-zip-file-containing-one-or-multiple-entries" /></p>
```
Comment (ealonso) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/view_import.jsp:85]: @claraizquierdo you should add an empty line between <p> and <liferay-frontend...

File: modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/view_import.jsp [hunk 2]
```diff
@@ -25,15 +25,20 @@ ImportDisplayContext importDisplayContext = new ImportDisplayContext(request, re
...
 	enctype="multipart/form-data"
 	name="fm"
+	onSubmit="<%= closeModal %>"
 >
 	<liferay-frontend:edit-form-body>
-		<liferay-ui:message key="import-help" />
+		<p><liferay-ui:message key="select-a-zip-file-containing-one-or-multiple-entries" /></p>
 
-		<a href="https://portal.liferay.dev/docs" target="_blank">
+		<a href=" https://learn.liferay.com/dxp/latest/en/site-building/creating-pages/adding-pages/exporting-and-importing-page-templates.html" target="_blank">
```
Comment (ealonso) [inline modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/view_import.jsp:41]: @claraizquierdo You can use <liferay-learn:message

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -14617,6 +14617,7 @@ select-a-transition=Select a Transition
 select-a-type=Select a Type
 select-a-unit=Select a unit, currently {0}.
 select-a-workflow=Select a Workflow
+select-a-zip-file-containing-one-or-multiple-entries=Select a Zip File containing one or multiple entries.
```
Comment (ealonso) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:14620]: @claraizquierdo Zip File, should be ZIP file, as we have in other language keys

File: modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/view_import.jsp [hunk 4]
```diff
@@ -25,15 +25,20 @@ ImportDisplayContext importDisplayContext = new ImportDisplayContext(request, re
 	<portlet:param name="portletResource" value="<%= portletDisplay.getId() %>" />
 </portlet:actionURL>
 
+<%
+String closeModal = "Liferay.Util.getOpener().Liferay.fire('closeModal');";
```
Comment (ealonso) [inline modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/view_import.jsp:29]: @claraizquierdo for this one, I think we can inline it, or move it to a script block

**Pull request closed without merging**